tasks/cachebust.js

```diff
@@ -67,7 +67,10 @@ module.exports = function(grunt) {
             var isCSS = (/\.css$/).test(filepath);
 
             findStaticAssets(markup, isCSS).forEach(function(reference) {
-                var filePath = (opts.baseDir ? opts.baseDir : path.dirname(filepath)) + '/';
+                // original code (doesn't handle css in a subdirectory with relative paths to assets):
+                // var filePath = (opts.baseDir ? opts.baseDir : path.dirname(filepath)) + '/';
+                // simpler new code. filepath already includes opts.baseDir so no need to check again:
+                var filePath = path.dirname(filepath) + '/';
```

Comment:
Just realised I didn't add the test to #106 - woops!! Will do it now (tiredness)